### ISSUE TITLE
Setup Ethereum multisig

### ISSUE THREAD

**Issue by litvintech** [cybercongress/congress#9]

**Comment by litvintech** [comment]
@21xhipster 
@litvintech 0x00B8Fe1A1A2b899418702e32A96E276Ff56A4D05
@hleb-albau 0x00725D89a2A2FB3B21Fd1035B579cbCDE4a0991b

**Comment by 21xhipster** [comment]
Contract in mainnet [`0xfc2dc6e4d7c1456d7b6602d61231d9a735854531`](https://etherscan.io/address/0xfc2dc6e4d7c1456d7b6602d61231d9a735854531)`

**Comment by 21xhipster** [comment]
Fail to use and verify self-deployed contract. It seems that the thing works better through [factory](https://etherscan.io/address/0x6e95c8e8557abc08b46f3c347ba06f8dc012763f)

The contract is [0x828e8db9cec907f1748a8ded74534ad49ca7790d](https://etherscan.io/address/0x828e8db9cec907f1748a8ded74534ad49ca7790d)

**Comment by 21xhipster** [comment]
@hleb-albau Cool. confirmed. Seems that the thing works. Waiting @litvintech

**Comment by 21xhipster** [comment]
@hleb-albau Hint: if you add our signatures (addresses) to parity addressbook it will be much more easier for you to understand proposals and transactions.

**Comment by hleb-albau** [comment]
@21xhipster Yeah, already done. it will be cool to share/manage/create team address book.

**Comment by litvintech** [comment]
@21xhipster @hleb-albau I have a problem with parity ui..
Problem is that parity UI don't open and operate with accounts
I tested parity stable 1.9.6 and 1.10.1 (with browser UI and latest release with stand-alone) bootstrapped from scratch (chain and keys both), tried ui in Chrome, Opera, Firefox and Safari - in all of them there is the same proplem, with all extension turned off
Sorry, but I need more time for fix that
<img width="1462" alt="screenshot 2018-04-23 14 08 21" src="https://user-images.githubusercontent.com/1690657/39123300-23de057c-4700-11e8-9099-213b3d1cb460.png">

**Comment by litvintech** [comment]
@21xhipster @hleb-albau I'm done with gnosis electrone multisig app
@21xhipster do we need any last test before start using wallet?

**Comment by litvintech** [comment]
@21xhipster @hleb-albau 
created two test tx with ID 10/11 for transfer 21X/ETH tokens to me/21xhipster

**Comment by 21xhipster** [comment]
# Summary
Failed to use default Gnosis-wallet CLI
Found a way to use Gnosis wallet UI to deploy gnosis contract using factory, send, and receive ethers and ERC-20 tokens
1 of 3 guys failed to use parity, thus use only Gnosis wallet remote config
